js/index.js

```diff
@@ -0,0 +1,36 @@
+const addToInputList = document.querySelector("#addName");
```

Comment:
Instead of using `document.querySelector`, you could have used [document.getElementById](https://developer.mozilla.org/en-US/docs/Web/API/Document/getElementById) which is more optimized performance-wise.

Reply:
Switching to "document.getElementsById" helped discard one variable declaration

<img width="553" alt="image" src="https://user-images.githubusercontent.com/35523688/225385432-1a80f646-30a4-449b-912b-71592078438e.png">

Reply:
This was discarded: 
<img width="244" alt="image" src="https://user-images.githubusercontent.com/35523688/225386090-1264ee40-be6a-4a86-b11c-6167503e1dd7.png">